util/logtools/make_timeline.ts

```diff
@@ -464,10 +450,11 @@ const assembleTimelineStrings = (
   let lastEntry: TimelineEntry = { time: lastAbilityTime.toString(), lineType: 'None' };
   if (fight !== undefined && fight.sealName !== undefined) {
     const zoneMessage = SFuncs.toProperCase(fight.sealName);
-    const tlString = `0.0 "--sync--" sync / 00:0839::${zoneMessage} will be sealed off/ window 0,1`;
+    const tlString =
+      `0.0 "--sync--" sync / 00:0839::${zoneMessage} will be sealed off/ window 100000,0`;
     assembled.push(tlString);
   } else {
-    assembled.push('0.0 "--sync--" sync /Engage!/ window 0,1');
+    assembled.push('0.0 "--sync--" sync /Engage!/ window 100000,0');
```

Comment:
Are you 100% sure this is doing what you intend here? Or more specifically, can you be 100% sure that this will work correctly if someone pulls early? I'm not familiar enough with `timeline.ts` to know, but my gut feeling is that if combat is already started before the timeline starts, a sync of X,0 will fail.

Reply:
Well, two things.  One you're correct that this is wrong.  If the end of the window doesn't extend past zero, it won't work at all.

However also, if somebody pulls early (probably) we shouldn't sync at all, since if some other sync has happened that's going to be more reliable.